docs/guidance/map/extensions.md

```diff
@@ -2,21 +2,19 @@
 
 OCDS provides a common core of [sections and building blocks](../../getting_started/building_blocks) for describing contracting processes.
 
-Many publishers will have additional data that they could publish. Instead of ignoring this data and leaving it unpublished, OCDS encourages publishers to collaborate on the creation of **extensions** to the standard. 
+Many publishers will have additional data that they could publish. Instead of ignoring this data and leaving it unpublished, OCDS encourages publishers to collaborate on the creation of **extensions** to the standard.
 
-Extensions are divided into two types:
+Some extensions are maintained by the Open Contracting Partnership (OCP) as part of the [standard governance process](../../governance/index), documented as part of the standard, and reviewed by the OCDS technical team with each version upgrade of OCDS. They are likely to be relevant to a large number of publishers and users.
```

Comment:
Extensions are no longer documented as part of the standard - we instead link to the Extension Explorer.

The following is implied by "as part of the governance process": "reviewed by the OCDS technical team with each version upgrade of OCDS"

The first sentence left it ambiguous whether all or some of OCP's extensions followed the governance process.

I moved a later sentence here so that it appeared in context.

The governance page now uses the phrase "standard development team"

```suggestion
Some extensions are maintained by the Open Contracting Partnership (OCP). Those that are likely to be relevant to a large number of publishers and users follow the [governance process](../../governance/index). The standard development team decides which extensions follow the governance process.
```